examples/workshop/04-load-following-battery.ipynb

```diff
@@ -0,0 +1,347 @@
+{
```

Comment:
For battery, we can specify init/min/max SOC from the top level config:

```python
technologies = {
    'pv': {
        'system_capacity_kw': solar_size_mw * 1000,
    },
    'wind': {
        'num_turbines': 10,
        'turbine_rating_kw': int(wind_size_mw * 1000 / 10)
    },
    'battery': {
        'system_capacity_kwh': battery_capacity_mw * 1000,
        'system_capacity_kw': battery_capacity_mw * 4 * 1000,
        'minimum_SOC': 20.0,
        'maximum_SOC': 100.0,
        'initial_SOC': 90.0
    },
    'grid': {
        'interconnect_kw': interconnection_size_mw * 1000
    }
}
```

Then we can remove the "Adjust Battery Parameters" section. The less that users have to manually dig into the PySAM system model, the better.

Reply:
Updated in 27f6b90b6dadbfb5911fb1788837d614e53227bf